script.js

```diff
@@ -61,3 +63,251 @@ const btn = document.querySelector('#hmbrg');
+    image: 'Img/SnapshootPortfolio.png',
+    alt: 'project 1',
+    technologies: ['html', 'css', 'javaScript'],
+    liveLink: 'https://petushka1.github.io/microverse-m1-w1-d3-portfolio-production',
+    sourceLink: 'https://github.com/petushka1/microverse-m1-w1-d3-portfolio-production',
+  },
+];
+
+const wrapper = document.querySelector('.wrapper');
+
+// card tempate
+div = document.createElement('div');
+div.className = 'gridContainer';
+wrapper.appendChild(div);
+const gridContainer = document.querySelector('.gridContainer');
+
+cardArr.forEach((item, i) => {
+  const card = document.createElement('div');
+  card.className = 'card round';
+  gridContainer.appendChild(card);
+
+  const snapshot = document.createElement('img');
+  snapshot.className = 'cardSnapshot';
+  snapshot.src = cardArr[i].image;
+  snapshot.alt = cardArr[i].alt;
+  card.appendChild(snapshot);
+
+  const cardContent = document.createElement('div');
+  cardContent.className = 'cardContent';
+  card.appendChild(cardContent);
+
+  const header = document.createElement('h2');
+  header.textContent = cardArr[i].name;
+  cardContent.appendChild(header);
+
+  const speclist = document.createElement('ul');
+  speclist.className = 'spec';
+  cardContent.appendChild(speclist);
+  const specArr = cardArr[i].spec;
+  for (let j = 0; j < specArr.length; j += 1) {
+    li = document.createElement('li');
+    li.className = 'specItem';
+    li.textContent = specArr[j];
+    speclist.appendChild(li);
+  }
+
+  const paragraph = document.createElement('p');
+  paragraph.textContent = cardArr[i].brief;
+  cardContent.appendChild(paragraph);
+
+  ul = document.createElement('ul');
+  ul.className = 'lng liststyle';
+  ul.style.marginBottom = '15px';
+  cardContent.appendChild(ul);
+  li = document.createElement('li');
+  li.className = 'tech';
+  let [first, second, third] = cardArr[i].technologies;
+  li.textContent = first;
+  ul.appendChild(li);
+  li = document.createElement('li');
+  li.className = 'tech';
+  li.textContent = second;
+  ul.appendChild(li);
+  li = document.createElement('li');
+  li.className = 'tech';
+  li.textContent = third;
+  ul.appendChild(li);
+  const details = document.createElement('button');
+  details.className = 'btn details';
+  details.textContent = 'See project';
+  details.name = 'details';
+  details.type = 'button';
+  details.id = cardArr[i].id;
+
+  // Card Button function
+  function openPopup(e) {
+    const j = e.target.id;
+    const popup = document.createElement('div');
+    popup.className = 'popup';
+    wrapper.appendChild(popup);
+
+    function closePopup() {
+      popup.style.display = 'none';
+    }
+
+    const popupContent = document.createElement('div');
+    popupContent.className = 'popupContent fontSmall';
+    popup.appendChild(popupContent);
+
+    img = document.createElement('img');
+    img.src = 'img/closeDark.svg';
```

Comment:
### **OPTIONAL**
- [ ] Please fix the broken link, for the pop-up closing button
It should be like this `img.src = _'Img/closeDark.svg';`

Reply:
can you advise on this? On my side, everything is working. Did you clone my repo?

Reply:
Yeah 
Change the image directory (folder) name in your code from `img` to `Img`